packages/cardhost/app/controllers/index.js

```diff
@@ -19,4 +22,15 @@ export default class IndexController extends Controller {
       doneScrolling: () => (this.selectedSection = sectionId),
     });
   }
+
+  @action
+  openCardNameDialog(title, model /*, evt*/) {
+    if (arguments.length > 2) {
+      this.cardModel = model;
+    }
+    if (arguments.length > 1) {
+      this.dialogTitle = title;
+    }
+    this.showDialog = true;
```

Comment:
Whenever you open a modal, it should receive focus. I am not sure if that is already handled in the dialog component or not.

See https://webaim.org/techniques/keyboard/

<img width="761" alt="Screen Shot 2020-01-29 at 10 37 42 AM" src="https://user-images.githubusercontent.com/16627268/73371299-7f2c9a80-4283-11ea-9ea0-de859228e0c9.png">

This page is also worth a read. It would be good to identify which things are the responsibility of this PR vs what should be implemented in the dialog compomnent: http://web-accessibility.carnegiemuseums.org/code/dialogs/

Reply:
This is maybe the first resource to look at: https://melsumner.gitbook.io/ember-component-patterns/messaging/modals

Reply:
cc @alexspeller since you are also working on modals

Reply:
@jenweber I did add an `{{autofocus}}` modifier that automatically gives the dialog focus when inserted. Any other changes that need to be made to the `dialog` component should probably be done in a separate PR.

As an aside...using the `<dialog>` HTML element is already somewhat problematic since it's not supported in Firefox or Safari. There is a `dialog` role, i.e. `<div role="dialog">` but I'm not sure whether or not that's more widely supported.